src/components/CRow.vue

```diff
@@ -94,7 +94,13 @@ export default {
         return null;
       }
       const endDatetime = this.endDatetime ? this.enqueuedDatetime : new Date();
-      return formatDistance(this.startedDatetime, endDatetime);
+      let expectedTime = null;
+      if (this.progress && this.progress > 0) {
+        expectedTime = ((new Date() - this.startedDatetime) * (1 - this.progress)) / this.progress;
+        expectedTime = format(expectedTime, 'm') + ' expected';
+      }
+      const executionTime = formatDistance(this.startedDatetime, endDatetime);
+      return executionTime + (expectedTime ? ',' + expectedTime : '');
```

Comment:
maybe something like:
15 min (12 min remaining) would be better

also add a progress column in %.